docs/myriad-configuration.md

```diff
@@ -26,6 +26,12 @@ frameworkName: MyriadAlpha
 # Myriad's mesos framework role.
 frameworkRole: someRoleName
 
+# User the Node Manager will run as (Defaults to user running the resource manager if absent,  necessary for remote distribution).
```

Comment:
What do you mean "necessary for remote distribution"? It defaults to a user, so it's always set anyway.
Nit: Also, there's a double space ' ' before "necessary".